<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_c_view</class>
 <widget class="QDialog" name="dlg_c_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_translator">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_evaluation">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>32</y>
     <width>156</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_origin">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>32</y>
     <width>156</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_translation">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>32</y>
     <width>157</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_emit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发布评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提交译文</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
